resolwe_bio/processes/import_data/bam.yml

```diff
@@ -116,3 +116,103 @@
         else
             echo "{\"proc.error\":\"BAI should have the same name as BAM with .bai extension\"}"
         fi
+
+- slug: upload-bam-primary
+  name: Primary hybrid BAM file
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/rnaseq:1.1.0
+    resources:
+      network: true
+  data_name: '{{ src.file|default("?") }}'
+  version: 0.0.1
+  type: data:alignment:bam:primary
+  flow_collection: sample
+  category: upload
+  persistence: RAW
+  description: >
+    Upload a primary mapping file in BAM format.
+  input:
+    - name: bam
+      label: Hybrid bam
+      type: data:alignment:bam
+      required: false
+      description: >
+        Primary bam will be uploaded to the same collection and sample where hybrid bam is.
```

Comment:
It will be appended to the same sample. The collection inclusion is just a consequence of the sample already being (or not) included in the collection(s).